packages/desktop-client/src/components/filters/FiltersMenu.js

```diff
@@ -205,62 +211,96 @@ function ConfigureField({
           {field === 'saved' && 'Existing filters will be cleared'}
         </View>
 
-        <Stack
-          direction="row"
-          align="flex-start"
-          spacing={1}
-          style={{ flexWrap: 'wrap' }}
-        >
-          {type === 'boolean'
-            ? [
-                <OpButton
-                  key="true"
-                  op="true"
-                  selected={value === true}
-                  onClick={() => {
-                    dispatch({ type: 'set-op', op: 'is' });
-                    dispatch({ type: 'set-value', value: true });
-                  }}
-                />,
-                <OpButton
-                  key="false"
-                  op="false"
-                  selected={value === false}
-                  onClick={() => {
-                    dispatch({ type: 'set-op', op: 'is' });
-                    dispatch({ type: 'set-value', value: false });
-                  }}
-                />,
-              ]
-            : ops.map(currOp => (
+        {type === 'boolean' ? (
+          <Stack
+            direction="row"
+            align="flex-start"
+            spacing={1}
+            style={{ flexWrap: 'wrap' }}
+          >
+            <OpButton
+              key="true"
+              op="true"
+              selected={value === true}
+              onClick={() => {
+                dispatch({ type: 'set-op', op: 'is' });
+                dispatch({ type: 'set-value', value: true });
+              }}
+            />
+            <OpButton
+              key="false"
+              op="false"
+              selected={value === false}
+              onClick={() => {
+                dispatch({ type: 'set-op', op: 'is' });
+                dispatch({ type: 'set-value', value: false });
+              }}
+            />
+          </Stack>
+        ) : (
+          [
```

Comment:
Since this is an array, the `key` prop is required (there’s now a warning thrown in the console in dev mode about this). However, since this is a fixed-size array you can swap in a fragment (`<>...</>`) to remove the warning without having to add a key.

Reply:
Got this sorted.